proxy/src/main/java/net/md_5/bungee/BungeeBossBar.java

```diff
@@ -0,0 +1,284 @@
+package net.md_5.bungee;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.EnumSet;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import net.md_5.bungee.api.boss.BarColor;
+import net.md_5.bungee.api.boss.BarStyle;
+import net.md_5.bungee.api.boss.BarFlag;
+import net.md_5.bungee.api.chat.BaseComponent;
+import net.md_5.bungee.api.connection.ProxiedPlayer;
+import net.md_5.bungee.chat.ComponentSerializer;
+import net.md_5.bungee.protocol.DefinedPacket;
+import net.md_5.bungee.protocol.ProtocolConstants;
+import net.md_5.bungee.protocol.packet.BossBar;
+
+@ToString
+@EqualsAndHashCode
+public class BungeeBossBar implements net.md_5.bungee.api.boss.BossBar
+{
+
+    @Getter
+    private BaseComponent[] title;
+    @Getter
+    private BarColor color;
+    @Getter
+    private BarStyle style;
+    @Getter
+    private float progress;
+    @Getter
+    private boolean visible;
+
+    private Set<ProxiedPlayer> players;
+    private EnumSet<BarFlag> flags;
+
+    @ToString.Exclude
+    private UUID uuid;
```

Comment:
Make it final and more initialization to the field, as this value may not change or it will break everything.